docs/EXAMPLES.md

```
@@ -0,0 +1,509 @@
+}
+```
+
+### Step 3: Run with Specific Target
+
+```bash
+.ralph/ralph.sh 15 --session auth-coverage --workflow test-coverage
+```
+
+### Expected Test Commits
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)